ui/docs/reference/workers/generic-worker/README.mdx

```diff
@@ -17,7 +17,7 @@ Depending on the operating system of the worker, and the generic-worker engine t
 Please see the following pages for guidance:
 
 <!-- BEGIN PAYLOAD LINKS -->
- * [Generic worker payload - multiuser, posix](/docs/reference/workers/generic-worker/multiuser-posix-payload)
+ * [Payload - multiuser, posix](/docs/reference/workers/generic-worker/multiuser-posix-payload)
```

Comment:
Is there any way we can fix this?

Reply:
I think the only _good way_ to fix this would be to fix the generator to just use "Generic worker payload" as the name as all of them as opposed to taking the name from the schema..

If you change the name of the other two schemas to also be `Payload`, then `GenericWorkerPayload` is not defined anymore and you run into a massive rabbit hole and/or chicken/egg problem while updating all the references. 🙃